### ISSUE TITLE
Stable release

### ISSUE THREAD

**Issue by mtmk** [nats-io/jwt.net#24]
### Proposed change

Are we ready for a stable?

### Use case

.

**Comment by rickdotnet** [comment]
The main feature I remember `nkeys.net` missing is XKey support. Not a knock on stability, imo. Personally, I'd push a stable release and then push an update when XKey support is added.

I'll look into the first two and see what I can help with there.

**Comment by mtmk** [comment]
> The main feature I remember `nkeys.net` missing is XKey support. Not a knock on stability, imo. Personally, I'd push a stable release and then push an update when XKey support is added.

I agree. I don't know of anyone asking for xkey we can add that later on if and when needed.

> I'll look into the first two and see what I can help with there.

Super! we'll hold back until then. thank you @rickdotnet

**Comment by mkobaly** [comment]
I am good with that plan. Currently we are using the pre-release and have not hit any other issues.

**Comment by Soundman32** [comment]
Would love to see a C#  example for auth callout https://natsbyexample.com/examples/auth/callout

**Comment by martinscheffler** [comment]
I am actually just implementing a nats callout with this library. For that I am missing method NatsJwt::DecodeAuthorizationRequestClaims(). Is that just an oversight, or is it not included for a reason?

**Comment by Soundman32** [comment]
@martinscheffler See https://github.com/nats-io/jwt.net/issues/29#issuecomment-2540927747
It's also missing the method you require, but it's an insecure workaround until they add the correct method.

**Comment by mtmk** [comment]
> I am actually just implementing a nats callout with this library. For that I am missing method NatsJwt::DecodeAuthorizationRequestClaims(). Is that just an oversight, or is it not included for a reason?

Thanks @Soundman32 we would like to implement anything missing. please feel free to open an issue or a PR in this repo.

**Comment by martinscheffler** [comment]
My current, very minimal nats callout implementation:

https://gist.github.com/martinscheffler/cca7105789553b151d900ae74ef94563

Another thing I noticed is: Why are methods like EncodeUserClaims() not static? Currently I have to create an instance of NatsJwt() to call it, after use I throw it away.
